controllers/sms-gateway.js

```diff
@@ -112,6 +112,8 @@ module.exports = {
       getOutgoing
     ], (err, [,,outgoingMessages]) => {
       if (err) {
+        console.error('There was an error processing the following SMS POST request');
+        console.error(JSON.stringify(req.body));
```

Comment:
I'm definitely not sure I want to do this. It was in fact you who convinced me to do it! I guess what you must have meant was "just for a little bit of time on production" and not in the actual codebase.